auth0/src/main/java/com/auth0/android/provider/WebAuthProvider.java

```diff
@@ -170,7 +169,7 @@ public void start(@NonNull Context context, @NonNull VoidCallback callback) {
             //Default values
             this.scheme = "https";
             this.ctOptions = CustomTabsOptions.newBuilder().build();
-            withResponseType(ResponseType.CODE);
+            this.values.put(KEY_RESPONSE_TYPE, RESPONSE_TYPE_CODE);
```

Comment:
Probably something we can hardcode inside the OauthManager class instead?

Reply:
We could add the response type in `OAuthManager` when we add the PKCE code challenge params. I kept them here since the values are used for other auth parameters, but can move them if you prefer.

Reply:
Given that the OAuthManager is the ultimate class that starts this, that one should be setting the "unchangeable" parameters. That would be, for example, the code flow required parameters. I'd prefer it to be moved internally.